alpa/device_mesh.py

```diff
@@ -996,17 +1010,21 @@ def _launch_xla_servers(self):
                 })
 
             # Launch the DaemonMoveWorker
-            node_resource = "node:" + self.host_info[i]["NodeManagerAddress"]
-            cls = ray.remote(resources={node_resource: 1e-3})(DaemonMoveWorker)
-            move_worker = cls.remote()
+            cls = ray.remote(DaemonMoveWorker)
+            move_worker = cls.options(
+                placement_group=placement_group,
+                placement_group_bundle_index=bundle_index).remote()
 
             # Launch the MeshHostWorker
-            cls = ray.remote(num_gpus=self.num_devices_per_host,
-                             resources={node_resource: 1e-3})(MeshHostWorker)
-            worker = cls.options(runtime_env={
-                "env_vars": env_vars
-            }).remote(self.server_address, self.num_hosts, i, self.mesh_id,
-                      move_worker, global_config.runtime_random_seed)
+            cls = ray.remote(num_cpus=0,
```

Comment:
The `MeshHostWorker` actually uses 1 CPU. To address your issue, can we use something like `num_cpus=1e-3`?